docs/docs/cmd/spo/tenant/tenant-settings-set.mdx

```diff
@@ -14,257 +14,494 @@ m365 spo tenant settings set [options]
-: Syncs privacy profile properties. Allowed values `true`, `false`
+: Syncs privacy profile properties. Allowed values `true`, `false`.
 
 `--ExcludedFileExtensionsForSyncClient [ExcludedFileExtensionsForSyncClient]`
-: Excluded file extensions for sync client. Array of strings split by comma
+: Excluded file extensions for sync client. Array of strings split by comma.
 
 `--AllowedDomainListForSyncClient [AllowedDomainListForSyncClient]`
-: Sets allowed domain list for sync client. Array of GUIDs split by comma. Example: `c9b1909e-901a-0000-2cdb-e91c3f46320a,c9b1909e-901a-0000-2cdb-e91c3f463201`
+: Sets allowed domain list for sync client. Array of GUIDs split by comma. Example: `c9b1909e-901a-0000-2cdb-e91c3f46320a,c9b1909e-901a-0000-2cdb-e91c3f463201`.
 
 `--DisabledWebPartIds [DisabledWebPartIds]`
-: Sets disabled web part Ids. Array of GUIDs split by comma. Example: `c9b1909e-901a-0000-2cdb-e91c3f46320a,c9b1909e-901a-0000-2cdb-e91c3f463201`
+: Sets disabled web part Ids. Array of GUIDs split by comma. Example: `c9b1909e-901a-0000-2cdb-e91c3f46320a,c9b1909e-901a-0000-2cdb-e91c3f463201`.
+
+`--DisabledModernListTemplateIds [DisabledModernListTemplateIds]`
+: Modern list template IDs to hide when creating a new list. An array of GUIDs split by a comma. Example: `c9b1909e-901a-0000-2cdb-e91c3f46320a,c9b1909e-901a-0000-2cdb-e91c3f463201`.
 
 `--DisableCustomAppAuthentication [DisableCustomAppAuthentication]`
-: Configure if ACS-based app-only auth should be disabled or not. Allowed values `true`, `false`
+: Configure if ACS-based app-only auth should be disabled or not. Allowed values `true`, `false`.
 
 `--CommentsOnListItemsDisabled [CommentsOnListItemsDisabled]`
-: Enables or disables comments on lists. Allowed values `true`, `false`
+: Enables or disables comments on lists. Allowed values `true`, `false`.
 
 `--EnableAzureADB2BIntegration [EnableAzureADB2BIntegration]`
 : Enables the preview for OneDrive and SharePoint integration with Azure AD B2B. Allowed values `true`, `false`. Azure AD one-time passcode needs to be enabled for this integration to work. For more information see [http://aka.ms/spo-b2b-integration](https://aka.ms/spo-b2b-integration).
 
+`--EnableAutoNewsDigest [EnableAutoNewsDigest]`
+: Enables or disables SharePoint auto news digest. This is an automated email to users about the latest news posts that are relevant to them, but that they may not have viewed yet. Allowed values `true`, `false`.
+
+`--AllowCommentsTextOnEmailEnabled [AllowCommentsTextOnEmailEnabled]`
+: Allows showing surrounding document context in email notifications when a user is mentioned. Allowed values `true`, `false`.
+
+`--CommentsOnFilesDisabled [CommentsOnFilesDisabled]`
+: Disables or enables commenting functionality on files. Allowed values `true`, `false`.
+
+`--DisableAddToOneDrive [DisableAddToOneDrive]`
+: Disables or enables `Add shortcut to OneDrive` feature in SharePoint document libraries.  Allowed values `true`, `false`.
```

Comment:
Can we also place this next to `HideSyncButtonOnODB` since these options are somewhat related?

```suggestion
: Disables or enables 'Add shortcut to OneDrive' feature in SharePoint document libraries.  Allowed values `true`, `false`.
```

Reply:
Pushed commit with required changes.